.github/workflows/validate-pull-request.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         env:
           JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+          
```

Comment:
Okay, so what you did revert was actually the spaces which we don't need. What we need here is a simple new line, do you see this?
![image](https://user-images.githubusercontent.com/19938236/218529513-3ab2f862-435c-4c9b-8838-c6b696ee6dc7.png)
Those are spaces that are not needed. I pushed the fix for this so we are not doing more rounds on this.